h/services/search_index/_queue.py

```diff
@@ -70,32 +70,31 @@ def add_where(self, tag, priority, where, force=False, schedule_in=None):
         self._db.execute(query)
         mark_changed(self._db)
 
-    def add(self, annotation_id, tag, schedule_in=None, force=False):
+    def add_between_times(self, start_time, end_time, tag, force=False):
```

Comment:
This is very confused because the two functions are similar, but I've just moved this above the other to make those after `add_where` alphabetical + the rename